README.md

```diff
@@ -1 +1,63 @@
-Gotta change this shithole of a readme later.
+exorium
+===
+exorium offers interaction commands, moderation, utility and other handy commands. Just wanna give someone a good hug or manage your server better? Then check us out!
+
+## Commmands
+### Utility
+|**Command**|**Description**|
+|---|---|
+|ping|See bot's latency to discord|
+|invite|Invite exorium to your server|
+|privacy|Read our privacy policy|
+|support|Get support with exorium|
+|id|Get a user's ID|
+|av|Get a user's avatar|
+|serverinfo|Get information about the server|
+|userinfo|See a user's info|
+|roleinfo|See a role's info|
+|servericon|Get the server's icon|
+|poll|Host a poll|
+|say|Say something with exorium|
+|stats|See exorium's statistics|
+|emote|Get emote info/url|
+|review|Review our bot!|
+|binary|Encode/decode binary|
+|base64|Encode/decode base64|
+|help|The help command|
+### Admin
+None so far.
```

Comment:
There are admin commands, they just aren't accessable for normal users as they are for bot admins.

Reply:
Oooh, bot admins, I thought server admins